flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionVertex.java

```diff
@@ -576,7 +576,7 @@ else if (numSources < parallelism) {
 	 */
 	public Execution resetForNewExecution(final long timestamp, final long originatingGlobalModVersion)
 			throws GlobalModVersionMismatch {
-		LOG.debug("Resetting execution vertex {} for new execution.", getTaskNameWithSubtaskIndex());
+		LOG.info("Resetting execution vertex {} to CREATED state for new execution.", getTaskNameWithSubtaskIndex());
```

Comment:
Currently we have log tracking vertex state transitions in most cases(through Execution::transitionState), but lack log of transition to CREATED state(it is directly set). I think this could be helpful to track vertex lifecycle. 
WDYT?

Reply:
I find `Resetting [...] to CREATED [...] for new execution` to be a bit verbose. I would be ok to change the log level to `INFO` so that it is consistent with. 
```
LOG.info("{} ({}) switched from {} to {}.", getVertex().getTaskNameWithSubtaskIndex(), getAttemptId(), currentState, targetState);
```

I would make it  a separate `[hotfix]` commit.

Reply:
Ok. Let's revert this change and do it later in a separate fix.